src/main/java/MorseCodeConverter.java

```diff
@@ -0,0 +1,51 @@
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+
+public class MorseCodeConverter {
+
+    private static final String[] code
+            = {".-", "-...", "-.-.", "-..", ".",
+            "..-.", "--.", "....", "..", ".---",
+            "-.-", ".-..", "--", "-.", "---",
+            ".--.", "--.-", ".-.", "...", "-",
+            "..-", "...-", ".--", "-..-", "-.--",
+            "--.."};
+    private final Map<String, Character> morseToEnglish = new HashMap<>();
+
+    public MorseCodeConverter() {
+        for (int i = 0; i < 26; i++) {
+            morseToEnglish.put(code[i], (char) ('a' + i));
+        }
+
+    }
+
+    public String toMorseCode(String englishInput) {
+        StringBuilder morseString = new StringBuilder();
+        String[] array = englishInput.split(" ");
+
+        for (String s : array) {
+            for (int i = 0; i < s.length(); i++) {
+                try {
+                    morseString.append(code[s.charAt(i) - 'a']).append(" ");
```

Comment:
Your string shouldn't have a " " at the end.

Reply:
how then do can you read the morse code

Reply:
*how do you read the morse code if its characters are not separated

Reply:
I mean a space at the end of the WHOLE Morse code string